src/__stories__/media-card-story.tsx

```diff
@@ -79,6 +83,7 @@ export const Default: StoryComponent<Args> = ({
             title={title}
             subtitle={subtitle}
             description={description}
+            icon={icon}
```

Comment:
rename to "asset" and copy the same usage as DisplayMediaCard story for example
![image](https://github.com/Telefonica/mistica-web/assets/6722153/655b0fec-959b-402a-a583-b4a46c3807d3)